### PR TITLE
[CP] RC2 Cherrypick

### PR DESCRIPTION
**Pull request by sambible**

### CODE REVIEW

File: lib/katello/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module Katello
-  VERSION = "4.15.0-master".freeze
+  VERSION = "4.15.0.rc1".freeze
```
Comment (chris1984) [inline lib/katello/version.rb:2]: Should this be rc2?

**Review by chris1984 — CHANGES_REQUESTED** [review]

File: lib/katello/version.rb (hunk 1 above)
Comment (sambible) [reply lib/katello/version.rb:2]: Yeah, probably. I might have not set it properly at some point

**Comment by ianballou** [comment]
Your first commit, [Release 4.15.0-rc1](https://github.com/Katello/katello/pull/11224/commits/971ea1940f6e56d3764f384297b75060d51cf67c) I think is pulled in as extra. That commit already exists in the 4.15 branch.

**Comment by ianballou** [comment]
I'd recommend a new commit for releasing RC2 with the updated version. Also might be worth regenerating the changelog in case anything was added.
I'm a little confused because the changelog currently in the 4.15 branch includes commits cherry-picked here... I'm not sure why that might've happened.

**Pull request closed without merging**